src/transformers/generation/flax_utils.py

```diff
@@ -283,6 +315,12 @@ def generate(
                 Number of beams for beam search. 1 means no beam search.
             decoder_start_token_id (`int`, *optional*):
                 If an encoder-decoder model starts decoding with a different token than *bos*, the id of that token.
+            suppress_tokens (`list`, *optional*):
+                Token ids to not sample
+            begin_suppress_tokens (`list`, *optional*):
+                Token ids to not sample during the first sample step
+            forced_decoder_ids (`list`, *optional*):
+                Sample indices and token ids to force sampling at beginning of generation
```

Comment:
```suggestion
            suppress_tokens  (`List[int]`, *optional*, defaults to `model.config.suppress_tokens`):
                A list of tokens that will be supressed at generation. The `SupressTokens` logit processor will set
                their log probs to `-inf` so that they are not sampled.
            begin_suppress_tokens  (`List[int]`, *optional*, defaults to `model.config.begin_suppress_tokens`):
                A list of tokens that will be supressed at the begining of the generation. The `SupressBeginTokens`
                logit processor will set their log probs to `-inf` so that they are not sampled.
            forced_decoder_ids (`List[List[int]]`, *optional*, defaults to `model.config.forced_decoder_ids`):
                A list of pairs of integers which indicates a mapping from generation indices to token indices that
                will be forced before sampling. For example, `[[1, 123]]` means the second generated token will always
                be a token of index 123.
```
Let's use a more detail docstring here (already written for `pt` and `tf`